androidClient/app/src/main/java/com/macbitsgoa/comrades/homefragment/HomeFragment.java

```diff
@@ -42,6 +47,19 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
         View view = inflater.inflate(R.layout.fragment_home, container, false);
+        ImageButton btnMyCourses = view.findViewById(R.id.btn_my_courses);
+        ImageButton btnMenus = view.findViewById(R.id.btn_menus);
+        ImageButton btnDocs = view.findViewById(R.id.btn_docs);
+        btnDocs.setOnClickListener(v -> CourseActivity.show(getContext(),
+                "-LJUx7EJ78rkt41kHNFs", "Imp. Docs"));
```

Comment:
What is `-LJUx7EJ78rkt41kHNFs` ?

Reply:
Imp. Docs is right now planned to be hardcoded for this release.This is the key of the course in firebase.

Reply:
What is Imp. Docs?

Reply:
Imp. Docs is a course used for storing important matetrials like Bulletins, timetables and other stuff people need regularly. I have here just hardcoded the reference to that course so, that its material list opens on button click